jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -433,16 +433,27 @@ def step900_collect_driver_artifacts() {
 
     // Always archive the simulation.log.gz from the run.
     archive "simulation-runner/results/**/*.log.gz"
-    dir('simulation-runner/results') {
-        deleteDir()
+}
+
+def create_params_file(archive_dir) {
+    def keys = params.keySet()
+    def values = params.values()
+    def param_string = ""
+    for (i = 0 i < keys.length; i++) {
+        param_string += "${keys[i]}: ${values[i]}\n"
```

Comment:
Do keySets and values ordered in the same way as each other?

Reply:
Good question. I should get the value based on the key in question.

Reply:
Updated.